src/tests/ftest/util/macsio_test_base.py

```diff
@@ -42,7 +42,14 @@ def __init__(self, *args, **kwargs):
     def setUp(self):
         """Set up each test case."""
         super(MacsioTestBase, self).setUp()
-        self.manager = Mpirun(None, subprocess=False, mpitype="mpich")
+
+        # Support using different job managers to launch the daos agent/servers
+        mpi_type = self.params.get("mpi_type", default="mpich")
+        if mpi_type == "openmpi":
+            self.manager = Orterun(None, subprocess=False)
+        else:
+            self.manager = Mpirun(None, subprocess=False, mpitype="mpich")
```

Comment:
(pylint-redefined-variable-type) Redefinition of self.manager type from job_manager_utils.Orterun to job_manager_utils.Mpirun